appveyor.yml

```diff
@@ -29,7 +29,7 @@ cache:
 install:
   # install SSL and php
   - cinst -y OpenSSL.Light
-  - cinst -y php
+  - cinst -y php --version 7.0.9
```

Comment:
This fixes the problem with Appveyor, there is an issue for that: http://help.appveyor.com/discussions/problems/5616-not-able-to-build-due-to-problem-in-chocolateyinstallps1

I subscribed to it, and will remove this again as soon as I get notified that it is fixed.